packages/dds/tree/src/test/shared-tree/fuzz/fuzzUtils.ts

```diff
@@ -34,8 +36,13 @@ export function runFuzzBatch(
 			saveOnSuccess: false, // Change to true to save successful runs.
 			filepath: path.join(__dirname, `fuzz-tests-saved-ops/ops_with_seed_${runSeed}`),
 		};
+		const savedInfo: SavedInfo | undefined = undefined;
+		// For using saved operations uncomment this and comment line above.
+		// const savedInfo: SavedInfo = {
+		// 	filepath: path.join(__dirname, `fuzz-tests-saved-ops/ops_with_seed_${runSeed}`),
+		// };
```

Comment:
Based on my other comments, these changes will go away. It is still nice to provide a commented example for loading from file like you did here though. So it would be nice if you could add a skipped test to topLevelFuzz.spec.ts (don't worry about targetedFuzz) that shows how to call the helper function with the path to make a generator, and pass that generator to the `performFuzzActions` function in topLevelFuzz.spec.ts.